src/VirtoCommerce.Platform.Data/ChangeLog/ChangeLogService.cs

```diff
@@ -98,9 +76,41 @@ public virtual async Task DeleteAsync(string[] ids)
                     repository.Remove(entity);
                 }
                 await repository.UnitOfWork.CommitAsync();
-                ChangeLogCacheRegion.ExpireRegion();
+                Reset();
             }
         }
+        #endregion
+
+        #region ILastModifiedDateTime Members
+        public DateTimeOffset GetLastModified(string entityName = null)
```

Comment:
Since have reset the last modified date of entities in DbTriggers, there is no sense to keep storing this code in ChangeLogService. Better to move this code to your own service.